<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPCarDialog</class>
 <widget class="QDialog" name="addPCarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add private car</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modl"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lcnr"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Owner Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owNa"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Owner Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owSu"/>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Owner Civic ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owID"/>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Ensured by:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ensu"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="addPCarButtonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addPCarButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addPCarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPCarButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addPCarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
